client/public/importCss.css

```diff
@@ -0,0 +1,12 @@
+@import 'css/aboutthisgame.css';
+@import 'css/achievements.css';
+@import 'css/addtocart.css';
+@import 'css/App.css';
+@import 'css/buttonLink.css';
+@import 'css/darkBox.css';
+@import 'css/infoPannel.css';
+@import 'css/ITGRTY.css';
+@import 'css/languages.css';
+@import 'css/pannelBar.css';
+@import 'css/socialPannel.css';
+@import 'css/titleLine.css';
```

Comment:
Did the legacy codebase not use CSS in JS or a CSS processor or anything?

Reply:
nope. just straight up css and it was all split up by files.

Reply:
so this was the easiest way to not screw up the front end